app/routes/profile.js

```diff
@@ -6,5 +6,8 @@ const { Route } = Ember;
 export default Route.extend(AuthenticatedRouteMixin, {
   titleToken() {
     return this.l10n.t('Profile');
+  },
+  model() {
```

Comment:
Why do you have to pass this in ? Not needed. You can directly access `authManager.currentUser` inside the template.